content/swiftui/concepts/color/color.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'Color'
+Description: 'Creates a color instance.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Color`** creates a color instance. `Color`s can be used as arguments, for example, to fill backgrounds and shapes or to apply color to text. They can also be used directly as [views](https://www.codecademy.com/resources/docs/swiftui/views), creating a rectangle area filled with their color, that adapts to fill the space available.
```

Comment:
Adjust this to include both the structure and standard color which are passed into certain modifiers.